lib/src/rrule_generator_widget.dart

```diff
@@ -0,0 +1,373 @@
+import 'dart:io';
```

Comment:
Using 'dart:io' for Platform.locale breaks the package on web. I think there should be an alternative that still works on web (worst case scenario keep using the same text delegate strategy).